airbyte-integrations/connectors/source-stripe/Dockerfile

```diff
@@ -14,5 +14,5 @@ ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
 
 
-LABEL io.airbyte.version=3.6.0
+LABEL io.airbyte.version=3.7.0
```

Comment:
Theres some conflicts, I think this will likely have to go to 3.9.0 or higher

Reply:
Updated